src/Sulu/Bundle/RouteBundle/Controller/RouteController.php

```diff
@@ -126,4 +148,20 @@ public function cdeleteAction(Request $request)
 
         return $this->handleView($this->view());
     }
+
+    private function generateUrlResponse(Request $request)
+    {
+        $resourceKey = $this->getRequestParameter($request, 'resourceKey');
+        $resourceKeyMapping = $this->resourceKeyMappings[$resourceKey] ?? null;
+
+        /** @var array $parts */
+        $parts = $this->getRequestParameter($request, 'parts', true);
+
+        $route = '/' . implode('-', $parts);
+        if ($resourceKeyMapping) {
```

Comment:
shouldnt we throw an exception if there is no mapping for the given resource key? 🤔

Reply:
wouldnt do this - the default behaviour seems good (as it is the same as for pages)